### PR TITLE
Add leveldb package

### PR DESCRIPTION
**Pull request by isaachier**

### CODE REVIEW

**Comment by isaachier** [comment]
Resolves #100.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2925
* https://travis-ci.org/ingenue/hunter/builds/311874754?utm_source=github_status&utm_medium=notification

File: examples/leveldb/main.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include <leveldb/db.h>
+
+int main()
+{
+    return 0;
```
Comment (chfast) [inline examples/leveldb/main.cpp]: Can you add anything that uses leveldb to check linking is working correctly?

Comment (isaachier) [reply examples/leveldb/main.cpp]: Lol TBH I never do that but I guess you are correct.

Comment (ruslo) [reply examples/leveldb/main.cpp]: Can we do this as a follow-up in next pull request? Tests almost finished (3 OSX job left) and I'm ready to merge this one. The rest looks good.

**Comment by isaachier** [comment]
Sure thing

File: examples/leveldb/main.cpp (hunk 1 above)
Comment (chfast) [reply examples/leveldb/main.cpp]: Yes.

**Comment by isaachier** [comment]
longest build ever...

**Comment by ruslo** [comment]
Waiting for free OSX machine, from https://www.traviscistatus.com/ :
osx-backlog

**Comment by chfast** [comment]
CircleCI provides fast macOS builds... for now.

**Comment by ruslo** [comment]
> CircleCI provides fast macOS builds... for now.

For free? https://circleci.com/pricing/#build-os-x

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2932
* https://travis-ci.org/ingenue/hunter/builds/312073399?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.180

**Comment by chfast** [comment]
>> CircleCI provides fast macOS builds... for now.
>
> For free? https://circleci.com/pricing/#build-os-x

Yes, I think you have to ask for it for open source projects. I'm using Circle CI 2.0 now: https://circleci.com/gh/ethereum/cpp-ethereum.

**Comment by ruslo** [comment]
> Yes, I think you have to ask for it for open source projects

The problem is that currently there is [step](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#ci-testing) when user run CI tests first on his fork to decide which toolchains have to be excluded. I guess it will mean that every user will have to ask Circle CI to add their Hunter's fork (?)

But I will keep this option in mind.

**Pull request merged**